packages/devtools_testing/lib/service_manager_test.dart

```diff
@@ -310,24 +312,28 @@ Future<void> runServiceManagerTestsWithDriverFactory(
     }
 
     test('all extension types', () async {
+      // TODO(kenz): Uncomment this once
```

Comment:
if commenting out blocks of code I think it is a little better to use `/*` or ideally use skip. as that keeps the diff cleaner.

Reply:
I commented instead of skipping so that we would still test String and Numeric extensions. Changed to /* */